File: BC_SENTDW-01/src/main/java/ar/gov/anses/Sentencias/Model/SentenciasModelImpl.java
package ar.gov.anses.Sentencias.Model;

import ar.gov.anses.Sentencias.DAO.SentenciasDaoImpl;
import ar.gov.anses.Sentencias.DTO.SentenciaDTO;
import ar.gov.anses.Sentencias.Domain.Sentencias;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

@Service
public class SentenciasModelImpl implements SentenciasModel{

    @Autowired
    private SentenciasDaoImpl sentenciasDao;

    @Override
    public SentenciaDTO findSentenciasById(Long sentenciaId){
        Sentencias sentencia = sentenciasDao.getById(sentenciaId);
        return new SentenciaDTO().mapToSentenciaDTO(sentencia);
    }

}

File: BC_SENTDW-02/Sentencias/Model/SentenciasModelImpl.cs
using System.Collections.Generic;
using PruebaBatch01.Sentencias.DAO;
using PruebaBatch01.Sentencias.DTO;
namespace PruebaBatch01.Sentencias.Model
{
    class SentenciasModelImpl
    {
        private  SentenciasDaoImpl sentenciasDao = new SentenciasDaoImpl();

        public List<SentenciaDTO> findSentenciasByIds(List<long> setenciasIds)
        {
            return sentenciasDao.findSentenciasByIds(setenciasIds);
        }

    }
}
